invisible_cities/icaros/krmap_functions_test.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pandas as pd
+from invisible_cities.core.core_functions import in_range, shift_to_bin_centers
+import itertools
+from new_icaromaps import create_NaN_map, med_fun, fit_function, map_3D_fits, merge_maps, include_coordinates, is_even
+from pytest import raises
+
+
+def test_empty_medfun():
+    empty_dst = pd.DataFrame(columns = ['DT', 'x', 'y', 'S2e'])
+    result = med_fun(empty_dst)
+
+    assert result.shape == (1, 5)
+    assert result['nevents'].sum() == 0
+    for col in ['median', 'sigma', 'median_error', 'sigma_error']:
+        assert pd.isna(result[col].iloc[0])
```

Comment:
```suggestion
        assert pd.isna(result[col].iloc[0]), f"{col} is not a nan"
```

this adds a bit of information in case of failure